tests/test_FFBbasis2D.py

```diff
@@ -138,13 +130,58 @@ def testRotate(self):
         y4 = basis.evaluate(v4)
 
         # Rotate 90
-        self.assertTrue(np.allclose(y1[0], y2[0], atol=1e-4))
+        self.assertTrue(np.allclose(y1[0], y2[0], atol=1e-5))
```

Comment:
Wanted to note from our conversation that if `np.rot90` indeed goes CCW, then we're comparing that to `rotate(-np.pi/2)`, which would be the opposing convention (making ours CW).

Don't if maybe worth making an issue to lock down exactly which convention we want to use and then check we're using it everywhere

Reply:
I've converted the code towards CCW. Now we just need to figure out why the complex rotation is reversed, but other than that, should be good. 😅